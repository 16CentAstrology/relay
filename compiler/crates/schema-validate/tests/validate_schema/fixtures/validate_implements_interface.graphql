interface Node {
  id: ID!
}

type Pet implements Node {
  id: ID # <--- Missing !
}

type Query {
  node: Node
}
