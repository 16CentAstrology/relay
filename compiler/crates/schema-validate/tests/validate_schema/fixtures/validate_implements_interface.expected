==================================== INPUT ====================================
interface Node {
  id: ID!
}

type Pet implements Node {
  id: ID # <--- Missing !
}

type Query {
  node: Node
}
==================================== OUTPUT ===================================
✖︎ Interface field 'Node.id' expects type 'ID!' but 'Pet.id' is of type 'ID'.

  validate_implements_interface.graphql:6:3
    5 │ type Pet implements Node {
    6 │   id: ID # <--- Missing !
      │   ^^
    7 │ }

  ℹ︎ The interface field is defined here:

  validate_implements_interface.graphql:2:3
    1 │ interface Node {
    2 │   id: ID!
      │   ^^
    3 │ }
